tests/macros/utils.js

```diff
@@ -154,11 +209,52 @@ function afterEachMacro(teardown) {
     });
 }
 
+/**
+ * Reads a generic fixture file from the `fixtures` directory.
+ *
+ * @param {string|string[]} filePath
+ *        A path to a file relative to the `fixtures` directory.
+ * @param {string|{encoding?:string,flag?:string}} [options]
+ *        Either the encoding for the result, or an object that contains the encoding
+ *        and an optional flag. If a flag is not provided, it defaults to `'r'`.
+ *
+ * @returns {string|Buffer}
+ */
+function readFixture(filePath, options) {
+    if (!Array.isArray(filePath)) {
+        filePath = [filePath];
+    }
+    let absolutePath = path.resolve(__dirname, 'fixtures', ...filePath);
+    if (options && typeof options !== 'string') {
```

Comment:
I don't see the necessity for this chunk of code. Why not just pass `options` through to the `readFileSync` call? It seems that you're making a copy of the object, but I'm not sure why that's necessary?

Reply:
It isn’t necessary, I just instinctively wrote this code.